### PR TITLE
build!: move LSP16 in its own package

### PR DESCRIPTION
**Pull request by YamenMerhi**
# What does this PR introduce?

Create package for LSP16 UniversalFactory standard.

### CODE REVIEW

File: docs/contracts/LSP16UniversalFactory/LSP16UniversalFactory.md
Comment (CJ42) [inline docs/contracts/LSP16UniversalFactory/LSP16UniversalFactory.md:1]: Need to update the path for the contract in the dodoc config file

File: packages/LSP16UniversalFactory/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# LSP Package Template
```
Comment (CJ42) [inline packages/LSP16UniversalFactory/README.md:1]: You forgot step 6 on this file.

image

Could you please update the title and put a basic description as just one sentence.

```suggestion
# LSP16 Universal Factory
```

File: packages/LSP16UniversalFactory/hardhat.config.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      outputSelection: {
+        '*': {
+          '*': ['storageLayout'],
+        },
+      },
+    },
+  },
+  packager: {
+    // What contracts to keep the artifacts and the bindings for.
+    contracts: [],
```
Comment (CJ42) [inline packages/LSP16UniversalFactory/hardhat.config.ts:116]: Maybe we might have to add the contracts to ship in this package, or we do it in separate PRs?

File: packages/LSP16UniversalFactory/remappings.txt
Comment (CJ42) [inline packages/LSP16UniversalFactory/remappings.txt:1]: You probably don't need this file I guess in the LSP16 package, since Foundry tests are not run in this package.

**Review by CJ42 — COMMENTED** [review]
Added review comments

**Review by richtera — APPROVED** [review]

**Review by richtera — APPROVED** [review]
